.github/actions/setup-pytorch/action.yml

```
@@ -45,8 +45,14 @@ runs:
       if: inputs.ref != ''
       shell: bash
       run: |
-        echo "PYTORCH_REPO=${{ inputs.repository }}" | tee -a "$GITHUB_ENV"
-        echo "PYTORCH_COMMIT_ID=${{ steps.commit-id.outputs.commit_id }}" | tee -a "$GITHUB_ENV"
+        if [[ ${{ inputs.repository == 'liangan1/pytorch' }} ]]; then
```

In bash, this is always true. I think this can be fixed with
```suggestion
        if [[ "${{ inputs.repository }}" = "liangan1/pytorch" ]]; then
```